runtime/src/parameters.rs

```diff
@@ -139,3 +148,97 @@ parameter_types! {
     // Vesting
     pub const MinVestedTransfer: Balance = CENT;
 }
+
+// Parameterized slow adjusting fee updated based on
+// https://research.web3.foundation/en/latest/polkadot/overview/2-token-economics.html#-2.-slow-adjusting-mechanism
+pub type SlowAdjustingFeeUpdate<R> =
+    TargetedFeeAdjustment<R, TargetBlockFullness, AdjustmentVariable, MinimumMultiplier>;
+
+#[cfg(test)]
+mod multiplier_tests {
+    use super::*;
+    use frame_support::{
```

Comment:
It is likely that more tests (in general, not only for the multipliers) will be added. I propose we create a tests module and start including tests for every logical component (i.e. transaction fees) within that module. This will unite all tests in one folder and we'll have a nice and easy to access overview, instead of having them dispersed around the runtime library code. You can have a look at Rikiddo to see how code is logically grouped in modules:

![image](https://user-images.githubusercontent.com/15378334/177790091-1c3eab11-cb36-4dc0-ab68-09f496f14814.png)